MAINTAINERS.yml

```diff
@@ -4448,6 +4448,11 @@ TDK Sensors:
   maintainers:
     - teburd
     - MaureenHelm
+    - afontaine-invn
```

Comment:
do we still need to have @teburd and @MaureenHelm  here? Maybe move them to collaborators?

Reply:
Works for me. Thanks for stepping up @afontaine-invn !

Reply:
Ok, you are now moved as collaborators.